modules/crm_core_match/src/Tests/FieldMatcherTest.php

```diff
@@ -71,6 +71,40 @@ public function testUnsupported() {
   /**
    * Test the text field.
    */
+  public function testName() {
+    $config = array(
+      'value' => array(
+        'operator' => '=',
+        'score' => 42,
+      ),
+    );
+    /** @var Contact $contact_needle */
+    $contact_needle = Contact::create(array('type' => 'individual'));
+    $contact_needle->set('name', 'Mr Gimeno Boomer');
```

Comment:
We need a test that covers multiple values matching.
It seems to me, when matching two different fields, they result in two disconnected queries and results in DefaultMatchEngine and the scores are not combined. Thus threshold is not compared properly. Followup!